catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogCapability.java

```diff
@@ -25,4 +25,18 @@ public CapabilityResult columnDefaultValue() {
         "The DEFAULT constraint for column is only supported since Hive 3.0, "
             + "but the current Gravitino Hive catalog only supports Hive 2.x.");
   }
+
+  @Override
+  public CapabilityResult caseSensitiveOnName(Scope scope) {
+    switch (scope) {
+      case SCHEMA:
+      case TABLE:
+      case COLUMN:
+        // Hive is case insensitive, see
+        // https://cwiki.apache.org/confluence/display/Hive/User+FAQ#UserFAQ-AreHiveSQLidentifiers(e.g.tablenames,columnnames,etc)casesensitive?
+        return CapabilityResult.unsupported("Hive is case insensitive.");
```

Comment:
Can you please add more logs here?

Reply:
Add more logs or comments?  More to describe what?

Reply:
I mean  the unsupported message, like you mentioned in comment.

Reply:
There is just one more link in the comment, should I put the link in the message?

Reply:
It' fine.